### DISCUSSION TITLE
官方播放器加入了一堆快捷键

### DISCUSSION THREAD

**Discussion by Geeyun-JY3** [the1812/Bilibili-Evolved#2231] [综合讨论 / General]
目前只在普通视频播放器搞了。
增加了开关弹幕、开关静音、投币、收藏的快捷键，改了点赞/取消点赞的快捷键，并支持了长按一键三连。奇怪的是还特地另外搞了一个 <kbd>R</kbd> 键只支持长按一键三连：
![image](https://user-images.githubusercontent.com/49396421/131286020-7f4fec81-dd04-42d7-bbd3-31100ee6e3de.png)

至于这个键位，开关弹幕、开关静音还挺正常，三连的三个相关快捷键好像就是直接用键盘左上角三（四）个字母。

**Comment by Geeyun-JY3** [comment]
目前 <kbd>W</kbd> 在全景视频里是无法调出投币对话框的。不知道官方怎么想的这个键位。

**Reply by the1812** [reply]
前几个还是好好按名称设的, 到三连这里变成搓技能似的(

**Reply by Geeyun-JY3** [reply]
可能要的就是搓技能那样连着的键位（

**Comment by ame1973** [comment]
有没有办法移除或禁用这些快捷键...

**Reply by the1812** [reply]
开脚本的快捷键扩展, 然后给官方的键位设个空的?
https://github.com/the1812/Bilibili-Evolved/blob/master/src/video/keymap/key-bindings.md

**Comment by Geeyun-JY3** [comment]
（好像是最近）加了两个新的快捷键：
- <kbd>Shift</kbd> + <kbd>1</kbd>：一倍速（正常倍速）
- <kbd>Shift</kbd> + <kbd>2</kbd>：二倍速

挺无语的，逐级调节视频播放速度的快捷键不做，搞了这么两个快捷键出来😅。
![image](https://user-images.githubusercontent.com/49396421/142436678-f252b68b-87ec-4a2d-8492-6ff68e57809f.png)
B 站哪找的人才😅
